.github/workflows/docker-cron.yaml

```diff
@@ -0,0 +1,93 @@
+name: Test Docker Image Building
+
+on:
+  push:
+    branches:
+      - "feat/add_image_building"
+  schedule:
+    # nightly tests
+    - cron: "0 0 * * *"
+
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: openfreeenergy/openfe
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Use dev tag for nightly builds
+        id: latest-version
+        run: |
+          VERSION=dev
+          echo $VERSION
+          echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Print Latest Version
+        run: echo ${{ steps.latest-version.outputs.VERSION }}
+
+      - name: Create fully qualified image registry path
+        id: fqirp
+        run: |
+          FQIRP=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ steps.latest-version.outputs.VERSION }}
+          echo "FQIRP=$FQIRP" >> $GITHUB_OUTPUT
+
+      - name: Print FQIRP
+        run: echo ${{ steps.fqirp.outputs.FQIRP  }}
+
+      - name: Build package
+        run: |
+          python3 -m pip install --upgrade build
```

Comment:
I saw you used that in the package cron, but I don't quite understand how installing `pipx` to install `build` is any better than using `pip` to install `build`. I use pipx mostly when I want to install a python app system wide in an isolated env. This doesn't _really_ matter so if we want to change it to using the same method as `package-tests.yaml` then I can

Reply:
I do it mostly so that I know I have a fully isolated environment for the build (especially when I then want to run an install + tests after). It really doesn't matter here tbh, I was more thinking out loud on if it would avoid needing to update wheel & build (won't really affect things tbh).

Reply:
Okay, I will keep it as is for now until the foot gun fires